File: abs/abc088_b/main.cc
#include <iostream>
#include <algorithm>

using namespace std;

int main() {
    int N; cin >> N;
    int a[N] = {0};
    for (int i = 0; i < N; ++i) cin >> a[i];
    sort(a, a + N, greater<int>());

    int alice = 0, bob = 0;
    for (int i = 0; i < N; ++i) {
        if (i % 2 == 0)
            alice += a[i];
        else {
            bob += a[i];
        }
    }
    cout << alice - bob << endl;
}

File: abs/abc088_b/main.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	var (
		N     int
		alice int
		bob   int
	)
	fmt.Scan(&N)

	a := make([]int, N)
	for i := 0; i < N; i++ {
		fmt.Scan(&a[i])
	}

	sort.Sort(sort.Reverse(sort.IntSlice(a)))

	for i := 0; i < N; i++ {
		if i%2 == 0 {
			alice += a[i]
		} else {
			bob += a[i]
		}
	}
	fmt.Println(alice - bob)
}
